backend/handlers/institution_children_request_handler.py

```diff
@@ -49,10 +49,13 @@ def put(self, user, request_key):
         institution_children = request.institution_requested_key.get()
         institution_children.set_parent(request.institution_key)
 
-        request.send_response_notification(user.current_institution, user.key, 'ACCEPT')
+        notification_id = request.send_response_notification(user.current_institution, user.key, 'ACCEPT')
```

Comment:
the send_response_notification method must be refactored, I think we should separate the creation of the notification and the sending in different methods when it is accepted the invite we call a method that creates and returns the id of the notification and when it is rejected we call send_response_notification